src/message/header/content_type.rs

```diff
@@ -67,6 +68,13 @@ impl FromStr for ContentType {
     }
 }
 
+#[cfg(feature = "mime03")]
```

Comment:
Could you also add a `doc(cfg(feature))` like this one https://github.com/lettre/lettre/blob/v0.10.0-rc.3/src/lib.rs#L117? It will make docs.rs show that this implementation is only available if the `mime03` feature is turned on